terminus/geometry/arc.py

```diff
@@ -221,6 +295,12 @@ def _angular_offset_for_point(self, point):
     def is_valid_path_connection(self):
         return self.radius() > 4.0
 
+    def almost_equal_to(self, other):
+        return self.circle().almost_equal_to(other.circle()) and \
+            self.start_point().almost_equal_to(other.start_point(), 5) and \
+            self.end_point().almost_equal_to(other.end_point(), 5) and \
+            self.angular_length() * other.angular_length() > 0
```

Comment:
If pep8 doesn't complain, can you make these column-aligned at the start of each line?